app/models/manageiq/providers/amazon/cloud_manager/provision/cloning.rb

```diff
@@ -2,9 +2,13 @@ module ManageIQ::Providers::Amazon::CloudManager::Provision::Cloning
   def do_clone_task_check(clone_task_ref)
     source.with_provider_connection(:sdk_v2 => true) do |ec2|
       instance = ec2.instance(clone_task_ref)
-      status   = instance.state.name.to_sym
-      return true if status == :running
-      return false, status
+      if instance.exists?
+        status   = instance.state.name.to_sym
```

Comment:
Don't need the extra padding if this isn't following https://github.com/ManageIQ/manageiq-providers-amazon/pull/604/files#diff-d41a9db602874dbc463a106f2cd10f97R4